README.md

```diff
@@ -76,6 +62,32 @@ Create a profile, review the sample use cases to familiarize yourself with the c
 
 You can now use all the functionalities of the extension.
 
+### Use Base Profiles
+
+Leverage existing base profiles with a token to log in to API Mediation Layer.
+
+Before using the base profiles functionality, ensure that you have [Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli.html) installed.
+
+**Follow these steps:**
+
+1. Open Zowe CLI and run the following command:
+
+   ```
+   zowe auth login apiml
+   ```
+
+2. Follow the instructions and provide all necessary information to create a base profile. 
```

Comment:
Follow the instructions to complete the login. 
A local base profile is created with your token.

See this for more information about the process--https://docs.zowe.org/stable/user-guide/cli-usingcli.html#how-token-management-works

Reply:
Should I include the link to this section?